README.md

```diff
@@ -101,6 +101,39 @@ If your environment requires a proxy, define the proxy properties in the JSON fi
   "proxy": "<proxy_host>:<proxy_port>"
 ```
 
+## Exception handling
+
+All the exceptions raised by the OneView inherits from HPOneViewException.
```

Comment:
Or something like this:

The OneView Python SDK handles all the communication with the OneView REST API and, in case of an error, a HPOneViewException, or one that inherits from it,  is fired.